src/api/public/apidocs-new/paths/source_project_name_package_name_binary_filename_attribute.yaml

```diff
@@ -0,0 +1,72 @@
+get:
+  summary: Get list of attributes and attribute values
+  description: |
+    Get a list of all attributes and their values
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/package_name.yaml'
+    - $ref: '../components/parameters/binary_filename.yaml'
+  responses:
+    '200':
+      description: OK. The request has succeeded.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/source/attributes.yaml'
+          example:
+            - name: AllowSubmitToMaintenanceRelease
+              namespace: OBS
+              binary: SLES-cd-DVD-x86_64
+            - name: OwnerRootProjectTest
+              namespace: OBS
+              binary: Ceph
```

Comment:
This is not correctly displayed by Swagger UI, but the reason is that you added the binary to the schema in the [previous PR](I know you added `binary` to the schema in the [other PR](https://github.com/openSUSE/open-build-service/pull/14009)) (which should be merged before this one).

![Screenshot 2023-03-17 at 18-50-39 Swagger UI](https://user-images.githubusercontent.com/2581944/225984710-53f38ee9-319c-4b8d-9840-1d543caf6ddb.png)